configs/raspberrypi2_wpe_ml_tv_defconfig

```diff
@@ -0,0 +1,100 @@
+BR2_PACKAGE_GST1_PLUGINS_GOOD_PLUGIN_AUDIOFX=y
+BR2_PACKAGE_GST1_PLUGINS_GOOD_PLUGIN_AUDIOPARSERS=y
+BR2_PACKAGE_GST1_PLUGINS_GOOD_PLUGIN_AUTODETECT=y
+BR2_PACKAGE_GST1_PLUGINS_GOOD_PLUGIN_DEINTERLACE=y
+BR2_PACKAGE_GST1_PLUGINS_GOOD_PLUGIN_ICYDEMUX=y
+BR2_PACKAGE_GST1_PLUGINS_GOOD_PLUGIN_ID3DEMUX=y
+BR2_PACKAGE_GST1_PLUGINS_GOOD_PLUGIN_INTERLEAVE=y
+BR2_PACKAGE_GST1_PLUGINS_GOOD_PLUGIN_MATROSKA=y
+BR2_PACKAGE_GST1_PLUGINS_GOOD_PLUGIN_SOUPHTTPSRC=y
+BR2_PACKAGE_GST1_PLUGINS_BAD_PLUGIN_MPEGTSDEMUX=y
+BR2_PACKAGE_GST1_PLUGINS_BAD_PLUGIN_VIDEOPARSERS=y
+BR2_PACKAGE_GST1_PLUGINS_BAD_PLUGIN_DVB=y
+BR2_PACKAGE_GST1_PLUGINS_BAD_PLUGIN_DASH=y
+BR2_PACKAGE_GST1_PLUGINS_BAD_PLUGIN_FAAD=y
+BR2_PACKAGE_GST1_PLUGINS_BAD_PLUGIN_HLS=y
+BR2_PACKAGE_GST1_PLUGINS_BAD_PLUGIN_OPUS=y
+BR2_PACKAGE_GST1_PLUGINS_BAD_PLUGIN_SMOOTHSTREAMING=y
+BR2_PACKAGE_GST1_PLUGINS_UGLY=y
+BR2_PACKAGE_GST1_PLUGINS_UGLY_PLUGIN_MPG123=y
+BR2_PACKAGE_GST1_LIBAV=y
+BR2_PACKAGE_GST_OMX=y
+BR2_PACKAGE_NETFLIX=y
+BR2_PACKAGE_NETFLIX_LIB=y
+BR2_PACKAGE_NETFLIX_KEYMAP="PartnerBridge_BCM.js"
+BR2_PACKAGE_NINJA=y
+BR2_PACKAGE_PLAYREADY=y
+BR2_PACKAGE_BITSTREAM_VERA=y
+BR2_PACKAGE_RPI_FIRMWARE=y
+# BR2_PACKAGE_EUDEV_ENABLE_HWDB is not set
+BR2_PACKAGE_RPI_USERLAND=y
+BR2_PACKAGE_DVB_APPS=y
+BR2_PACKAGE_DVB_WINTV_TUNER=y
+BR2_PACKAGE_WPEFRAMEWORK=y
+BR2_PACKAGE_WPEFRAMEWORK_PROVISIONPROXY=y
+# BR2_PACKAGE_WPEFRAMEWORK_COMMANDER is not set
+BR2_PACKAGE_WPEFRAMEWORK_LOCATIONSYNC=y
+BR2_PACKAGE_WPEFRAMEWORK_LOCATIONSYNC_URI="http://jsonip.metrological.com/?maf=true"
+BR2_PACKAGE_WPEFRAMEWORK_MONITOR=y
+BR2_PACKAGE_WPEFRAMEWORK_REMOTECONTROL=y
+BR2_PACKAGE_WPEFRAMEWORK_SNAPSHOT=y
+BR2_PACKAGE_WPEFRAMEWORK_TIMESYNC=y
+BR2_PACKAGE_WPEFRAMEWORK_TRACECONTROL=y
+BR2_PACKAGE_WPEFRAMEWORK_WEBKITBROWSER=y
+BR2_PACKAGE_WPEFRAMEWORK_WEBKITBROWSER_STARTURL="file:///www/index.html"
+BR2_PACKAGE_WPEFRAMEWORK_WEBKITBROWSER_USERAGENT="Mozilla/5.0 (Macintosh, Intel Mac OS X 10_11_4) AppleWebKit/602.1.28+ (KHTML, like Gecko) Version/9.1 Safari/601.5.17 WPE-Reference"
+BR2_PACKAGE_WPEFRAMEWORK_WEBKITBROWSER_MEMORYPRESSURE="databaseprocess:50m,networkprocess:80m,webprocess:250m,rpcprocess:80m"
+BR2_PACKAGE_WPEFRAMEWORK_WEBKITBROWSER_MEDIADISKCACHE=y
+BR2_PACKAGE_WPEFRAMEWORK_WEBKITBROWSER_DISKCACHE="512m"
+BR2_PACKAGE_WPEFRAMEWORK_WEBKITBROWSER_XHRCACHE=y
+BR2_PACKAGE_WPEFRAMEWORK_YOUTUBE=y
+BR2_PACKAGE_WPEFRAMEWORK_WEBSERVER=y
+BR2_PACKAGE_WPEFRAMEWORK_TVCONTROL=y
+BR2_PACKAGE_TVPLATFORM_LINUXTV=y
+BR2_PACKAGE_WPEFRAMEWORK_TVCONTROL_ATSC=y
+BR2_PACKAGE_WPEFRAMEWORK_PLUGINS_TVCONTROL_FREQUENCY_LIST="0"
+
+BR2_PACKAGE_WPEFRAMEWORK_DIALSERVER=y
+BR2_PACKAGE_WPEFRAMEWORK_NETFLIX=y
+BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_NETFLIX_AUTOSTART=y
+BR2_PACKAGE_WPEFRAMEWORK_PROVISIONING=y
+BR2_PACKAGE_WPEWEBKIT=y
+BR2_PACKAGE_LIBXKBCOMMON=y
+BR2_PACKAGE_ORC=y
+BR2_PACKAGE_ICU_USE_ICUDATA=y
+BR2_PACKAGE_SHARED_MIME_INFO=y
+BR2_PACKAGE_DROPBEAR=y
+BR2_PACKAGE_GESFTPSERVER=y
+BR2_TARGET_ROOTFS_INITRAMFS=y
+# BR2_TARGET_ROOTFS_TAR is not set
```

Comment:
Same here, does it hurt to enable TV control on all configs?
@albertd do you want a seperate config for the TV WPE enabled RPIs?

Reply:
@albertd , Could you please provide your suggestion on this ?